packages/editor-ui/src/plugins/codemirror/completions/nativesAutocompleteDocs/luxon.instance.docs.ts

```diff
@@ -622,12 +943,39 @@ export const luxonInstanceDocs: Required<NativeDoc> = {
 		toUTC: {
 			doc: {
 				name: 'toUTC',
+				description: i18n.baseText('codeNodeEditor.completer.luxon.instanceMethods.toUTC'),
 				section: 'edit',
 				docURL: 'https://moment.github.io/luxon/api-docs/index.html#datetimetoutc',
 				returnType: 'DateTime',
 				args: [
-					{ name: 'offset', type: 'number' },
-					{ name: 'opts', type: 'object' },
+					{
+						name: 'zone',
+						optional: true,
+						description: i18n.baseText(
+							'codeNodeEditor.completer.luxon.instanceMethods.toUTC.args.zone',
+						),
+						default: '"local"',
+						type: 'string',
+					},
+					{
+						name: 'opts',
+						optional: true,
+						description: i18n.baseText(
+							'codeNodeEditor.completer.luxon.instanceMethods.toUTC.args.opts',
+						),
+						type: 'Object',
+					},
+				],
+				examples: [
+					{
+						example:
+							"dt = '2024-01-01T00:00:00.000+02:00'.toDateTime()\ndt.setZone('America/Buenos_aires')",
```

Comment:
I'd expect caps, haven't checked though.

```suggestion
							"dt = '2024-01-01T00:00:00.000+02:00'.toDateTime()\ndt.setZone('America/Buenos_Aires')",
```